doc/common/chapter00.md

```diff
@@ -150,27 +151,28 @@ Following are a number of key architectural principles that apply to all Referen
 - Functional interfaces between workloads and workload management
 
 Due to the close alignment with ETSI GR NFV 002[3], those ETSI interfaces that are considered relevant (with notes where required) are included in the figure below.
 
-<p align="center"><img src="./figures/cntt-scope.png" alt="scope" title="Document Types" width="100%"/></p>
-<p align="center"><b>Figure 2:</b> Functional Scope of CNTT</p>
+<p align="center"><img src="./figures/anuket_scope.png" alt="scope" title="Document Types" width="100%"/></p>
+<!-- The source file of this figure is available at https://wiki.anuket.io/display/HOME/RA2+Diagrams -->
+<p align="center"><b>Figure 2:</b> Functional Scope of Anuket specifications</p>
 
 <a name="3.2"></a>
 ## Out of Scope Components
-While the nature of the CNTT might seem quite broad, the following areas are not at this time part of the scope of this effort.  
+While the nature of the Anuket project might seem quite broad, the following areas are not at this time part of the scope of this effort.  
 - Hardware specifications: beyond the abstracted high-level CPU, memory, network interface and storage elements.  The intention is to write the documents so they are general enough that any vendor hardware can be used in a conformant implementation without making significant changes to the model.
 - Workload specifications: Other than the API interfaces when they directly need to touch the workloads themselves, the intention is to assume the workload application is a black box that the cloud infrastructure is providing resources to.  The majority of interactions for lifecycle management of the workloads will be through the cloud infrastructure whenever possible.
-- Lifecycle Management of the CaaS Clusters: whilst a complete NFV-MANO solution would need to provide lifecycle management for the Kubernetes clusters it is using to deploy its CNFs, the CNTT doesn't describe the NFVO and VNFM parts, and therefore the management of the cluster(s) is not in scope, while the VIM and the lifecycle management of containers (by Kubernetes) is in scope.
-- Company specific requirements: The CNTT documents are designed to be general enough that most operators and others in the Open Source communities will be able to adapt and extend them to their own non-functional requirements.
+- Lifecycle Management of the CaaS Clusters: whilst a complete NFV-MANO solution would need to provide lifecycle management for the Kubernetes clusters it is using to deploy its CNFs, the Anuket specifications do not describe the NFVO and VNFM parts, and therefore the management of the cluster(s) is not in scope, while the VIM and the lifecycle management of containers (by Kubernetes) is in scope.
+- Company specific requirements: The Anuket specifications are designed to be general enough that most operators and others in the Open Source communities will be able to adapt and extend them to their own non-functional requirements.
```

Comment:
Later on "Open Source" is changed to all lower case. Should we be consistent and make changes throughout.